Meta/Lagom/Tools/CodeGenerators/LibWeb/WrapperGenerator/IDLParser.cpp

```diff
@@ -180,6 +180,13 @@ NonnullRefPtr<Type> Parser::parse_type()
         consume_whitespace();
 
     auto name = lexer.consume_until([](auto ch) { return !is_ascii_alphanumeric(ch) && ch != '_'; });
+
+    // check for "long long" type
+    if (name.equals_ignoring_case("long"sv)) {
+        if (lexer.consume_specific(" long"sv))
+            name = "long long";
+    }
```

Comment:
This needs to consume whitespace between both `long`s. Also, that comment isn't needed, this is obvious from the code.